src/UI/state/mapSlice.test.ts

```diff
@@ -320,6 +349,32 @@ describe('getOccurrenceData', () => {
     );
   });
 });
+describe('layerToggle', () => {
+  const fulfilled = {
+    type: getTileServerOverlays.fulfilled.type,
+    payload: [
+      {
+        name: 'testName',
+        sourceLayer: 'overlays',
+        sourceType: 'raster',
+      },
+      {
+        name: 'world',
+        sourceType: 'vector',
+        overlays: [{ name: 'testOverlay1' }, { name: 'testOverlay2' }],
+      },
+    ],
+  };
+
+  it('toggles the visibility of a layer', () => {
+    const initState = reducer(initialState, fulfilled);
+    const toggleLayerVisible = reducer(initState, layerToggle('testName'));
+    expect(
+      toggleLayerVisible.map_overlays.find((l: any) => l.name === 'testName')
+        ?.isVisible
+    ).toEqual(false);
```

Comment:
Can you add something toggling it back? It's important to check both ways, to ensure we're not just setting the value to false every time.